eks-log-collector.sh

```diff
@@ -190,72 +208,29 @@ pack()
 {
   try "archive gathered log information"
 
-  local tar_bin
-  tar_bin="`which tar 2>/dev/null`"
-  [ -z "${tar_bin}" ] && warning "TAR archiver not found, please install a TAR archiver to create the collection archive. You can still view the logs in the collect folder."
-
-  cd ${curdir}
-  ${tar_bin} -cvzf ${infodir}.tar.gz ${infodir} > /dev/null 2>&1
+  local TAR_BIN
+  TAR_BIN="$(command -v tar)"
 
-  ok
-}
-
-#
-# ---------------------------------------------------------------------------------------
-get_sysinfo()
-{
-  try "collect system information"
-
-  res="`/bin/uname -m`"
-  [ "${res}" = "amd64" -o "$res" = "x86_64" ] && arch="x86_64" || arch="i386"
-
-  found_file=""
-  for f in system-release redhat-release lsb-release debian_version; do
-    [ -f "/etc/${f}" ] && found_file="${f}" && break
-  done
-
-  case "${found_file}" in
-    system-release)
-      pkgtype="rpm"
-      if grep --quiet "Amazon" /etc/${found_file}; then
-        os_name="amazon"
-      elif grep --quiet "Red Hat" /etc/${found_file}; then
-        os_name="redhat"
-      fi
-      ;;
-    debian_version)
-      pkgtype="deb"
-      if grep --quiet "8" /etc/${found_file}; then
-        os_name="debian"
-      fi
-      ;;
-    lsb-release)
-      pkgtype="deb"
-      if grep --quiet "Ubuntu 14.04" /etc/${found_file}; then
-        os_name="ubuntu14"
-      fi
-      ;;
-    *)
-      fail
-      die "Unsupported OS detected."
-      ;;
-  esac
+  if [[ -z "${TAR_BIN}" ]]; then
+      warning "TAR archiver not found, please install a TAR archiver to create the collection archive. You can still view the logs in the collect folder."
+    else
+      ${TAR_BIN} --create --verbose --gzip --file "${HOME}"/ekslogsbundle_"${INSTANCE_ID}"_"$(date --utc +%Y-%m-%d_%H%M-%Z)".tar.gz --directory="${COLLECT_DIR}" . > /dev/null 2>&1
```

Comment:
Can we have a shorter name ? Something like instanceid_eks_log.tar.gz

Reply:
I added date, as there are times where the customer will re-run the log collect script a few times.  If we collect n+1 number of copies of the tar file, they all will be named:

instanceid_eks_log.tar.gz

As it stands, they will be named:
```
ekslogsbundle_i-05922013bdd159670_2018-09-10_2132-UTC.tar.gz
ekslogsbundle_i-05922013bdd159670_2018-09-10_2158-UTC.tar.gz
ekslogsbundle_i-05922013bdd159670_2018-09-10_2159-UTC.tar.gz
```

Let me work on making this shorter.

Reply:
How does:

```
eks_i-05922013bdd159670_2018-09-10_2159-UTC.tar.gz
```

Sound.  We still know:

- The service
- The instance ID
- Year, Month, Day, Time, Timezone when bundle was made.